### PR TITLE
Improve aws terraform configurations

### LINKED ISSUE #429: Ensure AWS resources in terraform examples are tagged
Currently not all AWS resources are tagged with special `kubernetes.io/cluster/${cluster.name} = ""` tag, that should be fixed

### PR DESCRIPTION
**Pull request by kron4eg**
Fixes #429

```release-note
Improve aws terraform:
* apply kubernetes tags
* allow fixating AMI
```

### CODE REVIEW

File: examples/terraform/aws/variables.tf [hunk 1]
```diff
@@ -19,7 +19,7 @@ variable "cluster_name" {
 }
 
 variable "aws_region" {
-  default     = "eu-west-3"
+  default     = "eu-central-1"
```
Comment (xmudrii) [inline examples/terraform/aws/variables.tf]: Why we changed back to `eu-central-1`? I think we had less number of flakes on `eu-west-3`.

File: examples/terraform/aws/output.tf [hunk 2]
```diff
@@ -45,9 +45,9 @@ output "kubeone_workers" {
   value = {
     # following outputs will be parsed by kubeone and automatically merged into
     # corresponding (by name) worker definition
-    fra1-a = {
+    pool-a = {
```
Comment (xmudrii) [inline examples/terraform/aws/output.tf]: Can this be potentially backwards incompatible? What if the user provisioned infrastructure using old scripts? Also, if we're doing this change, we should do it for all providers.

Comment (kron4eg) [reply examples/terraform/aws/output.tf]: We don't cover *example* terraform configurations with backward compatibility, and even if we do, we should cancel it. Those are only example show-cases, nothing more.

File: examples/terraform/aws/variables.tf (hunk 1 above)
Comment (kron4eg) [reply examples/terraform/aws/variables.tf]: The point was that automated clean-up of leftovers job is working in `eu-central-1`, but OK, reverted back to Paris.

**Pull request merged**